CONST_Makefile

```diff
@@ -89,7 +89,8 @@ NGEO_LIBS_JS_FILES += \
 	node_modules/bootstrap/dist/js/bootstrap.min.js \
 	node_modules/proj4/dist/proj4.js \
 	node_modules/d3/d3.min.js \
-	node_modules/typeahead.js/dist/typeahead.bundle.min.js
+	node_modules/typeahead.js/dist/typeahead.bundle.min.js \
+        node_modules/fuse.js/src/fuse.min.js 
```

Comment:
what not modify CONST_Makefile at all ? 

Can you please be more specific how to add an externs file to the build process through geoportailv3.mk ?